### PR TITLE
Routing changes

### PR DESCRIPTION
**Pull request by shrinathprabhu**
Resolves [AR-4007](https://team-1624093970686.atlassian.net/browse/AR-4007).

## Changes

- Add `:appId` params to the routes for dashboard, configure and users
- Rename /app to /apps
- Pass params while doing router.push

## Checklist

- [x] The branch name follows the format: `developer/AR-XXX-issue-name`.
- [x] The changes have been tested locally.

### CODE REVIEW

File: src/pages/AppConfigure.vue [hunk 1]
```diff
@@ -42,7 +42,10 @@ let currentConfig = getAppConfigRequestBody()
 
 function switchTab(tab: ConfigureTab) {
   currentTab.value = tab.type
-  router.replace({ name: `${tab.label}Settings` })
+  router.replace({
```
Comment (ajithranka) [inline src/pages/AppConfigure.vue:45]: Change to `router.push`. We want to user to be able to use the back button when navigating the configure pages.

File: src/router/index.ts [hunk 2]
```diff
@@ -55,28 +55,30 @@ const routes: RouteRecordRaw[] = [
     redirect: '/login',
   },
   {
-    path: '/app',
-    redirect: appStore.appId ? '/app/dashboard' : '/app/create',
+    path: '/apps',
+    redirect: appStore.appId
```
Comment (ajithranka) [inline src/router/index.ts:59]: Is the plan to remove this later when add the "manage apps" page?

**Review by ajithranka — APPROVED** [review]

File: src/router/index.ts (hunk 2 above)
Comment (shrinathprabhu) [reply src/router/index.ts:59]: Yeah we'll remove redirect and place the component for manage apps

File: src/pages/AppConfigure.vue (hunk 1 above)
Comment (shrinathprabhu) [reply src/pages/AppConfigure.vue]: Resolved at https://github.com/arcana-network/developer-dashboard/pull/123/commits/a646f74a4dd17b878ec757822ec6e8376a3eeb1f

**Review by karthik-durai — APPROVED** [review]

**Pull request merged**